cabal-testsuite/PackageTests/ForeignOptsC/foreign-opts-c.cabal

```diff
@@ -0,0 +1,14 @@
+cabal-version:       2.2
```

Comment:
make a top directory for these tests, maybe `PackageTests/FFI/ForeignOptsC` and `..Cxx`. As you may notice, the top level is already quite crowded, lets not have individual tests there.